READMEs/marketplace-flow.md

```diff
@@ -276,19 +276,11 @@ print(f"order_tx_id = '{order_tx_id}'")
 
 # Bob downloads. If the connection breaks, Bob can request again by showing order_tx_id.
 file_path = ocean.assets.download_asset(
-    asset,
-    service.service_endpoint,
-    bob_wallet,
-    './',
-    order_tx_id
-)
-
-file_path = ocean.assets.download(
-    asset.did,
-    service.index,
-    bob_wallet,
-    order_tx_id,
-    destination='./'
+    asset=asset,
+    provider_uri=service.service_endpoint,
```

Comment:
no, again, the provider_uri and service_endpoint are different things. Pass in the service and get what you need (id and endpoint) from there.

Reply:
I forgot to delete from here